### ISSUE TITLE
display: grid;

### ISSUE THREAD

**Issue by Giftykik** [android-js/androidjs#300]
the apk app cant show grid that i made in my CSS.

in my html i use  div id="myid"
in css
#myid{
display: grid;
}
in browser firefox all work
on android i got nothing

can you please help me with this problem

**Comment by ftk789** [comment]
Did you link the css file in the HTML?

**Comment by Giftykik** [comment]
> Did you link the css file in the HTML?

Yes I do I had testing it with link and with inner <style> tag. steal not working (

**Comment by ftk789** [comment]
> 

Can you show me your code? Because it clearly works for me.

**Comment by Giftykik** [comment]
<html>
<head>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<style>
	
body{ background: #420000; }
#R { background-color: #00ff00; }
.target { background-color: #b4178d; }
.tainer {background-color: #174eb4;}

#grd{
	display: grid;
	position: absolute;
	width: 100%;
	height: 100%;
	grid-template-rows: 25px repeat(6,6fr) 25px ;
	grid-template-columns: 25px repeat(5,5fr) 25px ;
}
</style>

<body>

	<div id="grd">
		<div class="target" style="grid-area: 3 / 4;" name="trg 3 / 4"></div>
		<div class="tainer" style="grid-area: 4 / 4;" name="4 / 4"></div>
		<div id="R" style="grid-area: 6 / 4;" name="6 / 4"></div>
	</div>

</body>

</html>

**Comment by Giftykik** [comment]
`<html>
<head>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<style>
	
body{ background: #420000; }
#R { background-color: #00ff00; }
.target { background-color: #b4178d; }
.tainer {background-color: #174eb4;}

#grd{
	display: grid;
	position: absolute;
	width: 100%;
	height: 100%;
	grid-template-rows: 25px repeat(6,6fr) 25px ;
	grid-template-columns: 25px repeat(5,5fr) 25px ;
}
</style>

<body>

	<div id="grd">
		<div class="target" style="grid-area: 3 / 4;" name="trg 3 / 4"></div>
		<div class="tainer" style="grid-area: 4 / 4;" name="4 / 4"></div>
		<div id="R" style="grid-area: 6 / 4;" name="6 / 4"></div>
	</div>

</body>

</html>`

**Comment by Giftykik** [comment]
<html>
<head>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<style>
	
body{ background: #420000; }
#R { background-color: #00ff00; }
.target { background-color: #b4178d; }
.tainer {background-color: #174eb4;}

#grd{
	display: grid;
	position: absolute;
	width: 100%;
	height: 100%;
	grid-template-rows: 25px repeat(6,6fr) 25px ;
	grid-template-columns: 25px repeat(5,5fr) 25px ;
}
</style>

<body>

	<div id="grd">
		<div class="target" style="grid-area: 3 / 4;" name="trg 3 / 4"></div>
		<div class="tainer" style="grid-area: 4 / 4;" name="4 / 4"></div>
		<div id="R" style="grid-area: 6 / 4;" name="6 / 4"></div>
	</div>

</body>

</html>

**Comment by Giftykik** [comment]
i am using android 5.0.2

**Comment by Giftykik** [comment]
full code
https://github.com/Giftykik/androidjs_troubles/blob/main/grid

**Comment by Giftykik** [comment]
i dont even need server part i need only client part

**Comment by ftk789** [comment]
![image](https://github.com/android-js/androidjs/assets/57598907/51d48c13-7d41-4efd-b62a-f9804e9d4803)
It works perfectly fine! Unless I'm miss understanding you.

**Comment by Giftykik** [comment]
yes... but on my device i got only background... wtf.. my android 5.0.2

**Comment by Giftykik** [comment]
> ![image](https://user-images.githubusercontent.com/57598907/242288189-51d48c13-7d41-4efd-b62a-f9804e9d4803.png) It works perfectly fine! Unless I'm miss understanding you.

but this emulator, what if you test it on real device?

**Comment by ftk789** [comment]
> > ![image](https://user-images.githubusercontent.com/57598907/242288189-51d48c13-7d41-4efd-b62a-f9804e9d4803.png) It works perfectly fine! Unless I'm miss understanding you.
> 
> but this emulator, what if you test it on real device?

Same results, I think the issue here is something else, See if probably your main.js has anything in it, And see if there's wrong files in wrong places.

**Comment by Giftykik** [comment]
I just copy index.html in views dir and do not change main.js
do i have to wright something in main.js?

**Comment by Giftykik** [comment]
i was trying even delete main.js and the same result (

**Comment by ftk789** [comment]
> i was trying even delete main.js and the same result (

I'll give you somethign that might help, There's a repo that has sample apps, Get one of them and copy and paste your stuff there, Should work.

**Comment by Giftykik** [comment]
Thank you very much! How i can get it?

I think its SDK version trouble cos other phone work normaly, only my Android 5.0.2 do not. May be it is too old...
